### DISCUSSION TITLE
Semidefinite Program Example

### DISCUSSION THREAD

**Discussion by tzheng8** [cvxpy/cvxpy#1630] [Q&A] [answered]
Hi, has anyone worked with Semidefinite Program, especially the tutorial?  https://www.cvxpy.org/examples/basic/sdp.html

I am very confused by the Example, where C, A should be all symmetric matrices by definition.

**Comment by SteveDiamond** [comment, marked as answer]
You're right. We should change the example so C and A are symmetric. Sorry about that!
